File: Inca Runner/Assets/2dinfiniterunner/Scripts/CSharp/flyerScore.cs
using UnityEngine;
using System.Collections;

public class flyerScore : MonoBehaviour {

	public AudioClip pointSound;
	
	private int score = 0;
	
	void Start () {
		updateScore();
	}
	
	void addPoint () {
		score += 1;
		GetComponent<AudioSource>().PlayOneShot(pointSound);
		updateScore();
	}
	
	void updateScore () {
		GetComponent<GUIText>().text = "SCORE: "+score.ToString();
		var getScore = PlayerPrefs.GetInt("flyHighScore");
		if(score > getScore){
			PlayerPrefs.SetInt("flyHighScore", score);
		}
	}
}

File: Inca Runner/Assets/2dinfiniterunner/Scripts/Javascript/flyerScore.js
#pragma strict

var pointSound:AudioClip;

private var score:int = 0;

function Start () {
	updateScore();
}

function addPoint () {
	score += 1;
	GetComponent.<AudioSource>().PlayOneShot(pointSound);
	updateScore();
}

function updateScore () {
	GetComponent.<GUIText>().text = "SCORE: "+score.ToString();
	var getScore = PlayerPrefs.GetInt("flyHighScore");
	if(score > getScore){
		PlayerPrefs.SetInt("flyHighScore", score);
	}
}